Modules/TestQuestionPool/classes/tables/class.ilUnitCategoryTableGUI.php

```diff
@@ -34,10 +34,12 @@ public function __construct(ilUnitConfigurationGUI $controller, $cmd)
         $this->setDefaultOrderDirection('category');
         $this->setDefaultOrderDirection('ASC');
 
-        if ($this->getParentObject()->isCRUDContext()) {
-            $this->addMultiCommand('confirmDeleteCategories', $this->lng->txt('delete'));
-        } else {
-            $this->addMultiCommand('confirmImportGlobalCategories', $this->lng->txt('import'));
+        if ($DIC->rbac()->system()->checkAccess('write', $DIC->workflowEngine()->internal()->request()->getRefId())) {
```

Comment:
$DIC->workflowEngine()->internal()->request() is wrong here in the QuestionPool, use $DIC->testQuestionPool()->internal()->request() instead, please.